src/chats/components/conversation/messageBubbles/ReactionChip.tsx

```diff
@@ -4,20 +4,22 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import React, { ReactElement, useMemo } from 'react';
+import React, { ReactElement, useCallback, useMemo, useState } from 'react';
 
 import { Avatar, Container, Padding, Tooltip } from '@zextras/carbonio-design-system';
-import { map, size } from 'lodash';
+import { includes, map, size } from 'lodash';
 import styled from 'styled-components';
 
 import useAvatarUtilities from '../../../../hooks/useAvatarUtilities';
+import { getXmppClient } from '../../../../store/selectors/ConnectionSelector';
+import { getUserId } from '../../../../store/selectors/SessionSelectors';
 import { getUserName } from '../../../../store/selectors/UsersSelectors';
 import useStore from '../../../../store/Store';
 
-const CustomContainer = styled(Container)`
+const CustomContainer = styled(Container)<{ $animation: boolean }>`
 	border-radius: 1.25rem;
 	font-size: 0.9rem;
-	cursor: default;
+	cursor: pointer;
 	animation: bounceIn 0.4s ease-in-out;
```

Comment:
Based on new check you've done at line 40 this must be removed

Reply:
I have to keep this one too because this makes the animation happen when the chip is mounted, the other check makes animation happen when user clicks on it